docs/configuration.md

```diff
@@ -0,0 +1,23 @@
+# TesseraCT Configuration
+
+## Flags
+
+### Publication Awaiter
+
+The `enable_publication_awaiter` flag enables the publication awaiter, which waits for the checkpoint to be published before returning. As the default checkpoint interval is 10 seconds, it is recommended to update the `checkpoint_interval` flag to a lower value for performance. Note that the [maximum rate of writes to the checkpoint file is one write per second on Google Cloud Storage](https://cloud.google.com/storage/quotas#objects). Tessera enforces the minimum permitted checkpoint interval check during the TesseraCT application initialization process.
```

Comment:
Do you want to imply that this flag should be set to value higher than 1 write per second? If yes, say it directly (and we could also implement this in the code).

Reply:
The Tessera code will exit if the flag value is set to lower than 1000ms for AWS and 1200ms for GCP. I can add that here but it could change in the future.

Reply:
Added the checkpoint interval section with a Tessera limit table.